test/functional/transactions/0.tx.js

```diff
@@ -0,0 +1,214 @@
+'use strict';
+
+var _ = require('lodash');
+var crypto = require('crypto');
+
+var node = require('../../node.js');
+var http = require('../../common/httpCommunication.js');
+var sendTransaction = require('../../common/complexTransactions.js').sendTransaction;
+
+var signedTransactionFromGenesis = {
```

Comment:
@Isabello told me to extract it from there. Shall we add it to `node.js` general file?